src/main/java/com/mcmiddleearth/guidebook/command/GuidebookSet.java

```diff
@@ -187,36 +139,36 @@ private void saveData(CommandSender cs, InfoArea area) {
 
      @Override
      public void confirmed(Player player) {

```

Comment:
The main change -  creating a new region instead of a whole new area, so that nothing about the area is lost